lutris/gui/lutriswindow.py

```diff
@@ -537,13 +537,13 @@ def get_service_media(self, icon_type):
             return medias[icon_type]()
         return medias[service.default_format]()
 
-    def update_revealer(self, game=None):
-        if game:
+    def update_revealer(self, games=None):
+        if games:
             if self.game_bar:
                 self.game_bar.destroy()
-
-            self.game_bar = GameBar(game, self.application, self)
-            self.revealer_box.pack_start(self.game_bar, True, True, 0)
+            if len(games) == 1 and games[0]:
+                self.game_bar = GameBar(games[0], self.application, self)
+                self.revealer_box.pack_start(self.game_bar, True, True, 0)
```

Comment:
Need an else: around here. If you do not replace self.game_bar, set it to None at the least.

Reply:
Isn't it already destroyed in the above if?

Reply:
No. The field 'self.game_bar' is not the same thing as the game bar; it is but a reference to it, and you've left a reference to a destroyed control. That's not a great thing to do; clear the field too, via 'self.game_bar = None'

Reply:
Ok